src/test/java/stringcalculator/StringValidation.java

```diff
@@ -1,13 +1,14 @@
 package stringcalculator;
 
 import java.util.Arrays;
+import java.util.regex.Pattern;
 
 public class StringValidation {
 
-    private final DelimiterParser delimiterParser;
+    private final String text;
 
-    public StringValidation() {
-        this.delimiterParser = new DelimiterParser();
+    public StringValidation(String text) {
+        this.text = text;
```

Comment:
```suggestion
    public StringValidation(String text) {
        validateNum(text);
        this.text = text;
```

와 같이 표현해 볼 수 있지 않을까요?